charts/kube-otel-stack/values.yaml

```diff
@@ -82,8 +82,20 @@ tracesCollector:
         send_batch_size: 1000
         timeout: 1s
         send_batch_max_size: 1500
+      k8sattributes:
+        passthrough: false
+        extract:
+          metadata:
+            - k8s.cluster.name
+            - k8s.namespace.name
+            - k8s.pod.name
+            - k8s.pod.uid
+            - k8s.node.name
       resource:
         attributes:
+        - key: lightstep.helm_chart
+          value: kube-otel-stack
+          action: insert
         - key: collector.name
           value: "${KUBE_POD_NAME}"
```

Comment:
I think `KUBE_POD_NAME` no longer exists. So I think this should be renamed to `${OTEL_K8S_POD_NAME}` otherwise the `container.name` attribute will be missing